spec/README.md

```diff
@@ -79,12 +79,16 @@ A reference to a _term_ looks like this.
 > The provisions of the stability policy are not in effect until
 > the conclusion of the technical preview and adoption of this specification.
 
-Updates to this specification will not change
-the syntactical meaning, the runtime output, or other behaviour
-of valid messages written for earlier versions of this specification
-that only use functions defined in this specification.
-Updates to this specification will not remove any syntax provided in this version.
-Future versions MAY add additional structure or meaning to existing syntax.
+Updates to this specification shall not make any well-formed or valid _message_ invalid.
+Updates to this specification shall not remove any syntax provided in this version.
+Future versions might add additional structure or meaning to existing syntax.
+
+Updates to this specification shall not change the syntactical meaning
+of any syntax defined in this specification except for that syntax marked as
+"reserved for future standardization".
```

Comment:
Isn't this already covered by the following?
> Updates to this specification will not reserve or assign meaning to
any character "sigils" except for those in the `reserved` production.

Reply:
I'm trying here to account for the original text's "syntactical meaning" promise, which I think is important (e.g. `.input` will not suddenly mean something different from an input declaration, `.match` will always be a selector, etc.). The `reserved-keyword` and `reserved-annotation` promises are separate and the promise not to define even more sigils (`reserved-annotation-start`) are yet more.

The one you quote should really mention the `reserved-annotation-start` production.